### PR TITLE
Storefront improve ui

### PR DESCRIPTION
**Pull request by skarpyak**
Description:
updated ui according to new mockups 

Linting:

- [x] No linting errors

Tests:

- [x] Smoke tests (mvn clean install -Dcucumber.options="--tags @smoketest" from test)
- [x] Manual tests

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

File: src/components/carousel.homepage.less [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+}
+.slick-initialized .slick-slide{
+  display:block
+}
+.slick-vertical .slick-slide{
+  display:block;
+  height:auto;
+  border:1px solid transparent
+}
+.slick-arrow.slick-hidden{
```
Comment (shaunmaharaj) [inline src/components/carousel.homepage.less]: Need to also hide `slick-arrow`:

```
.slick-slider slick-next,
.slick-slider slick-previous {
display: none;
}
```

**Comment by shaunmaharaj** [comment]
Can we please update the following styles in appheader.main.less as well to reduce the padding for the new header elements in mobile view?

This will line up better with the logo in mobile:
image
```
.app-header .main-container .search-container,
.app-header .main-container .login-container,
.app-header .main-container .cart-link-container {
    padding-top: 15px;
```

File: src/components/appheader.main.less [hunk 2]
```diff
@@ -113,50 +127,75 @@
...
       .central-container {
-        .horizontal-menu {
-          display: none;
-        }
+        display: none;
       }
 
       .search-container {
-        display: none;
+        display: inline-block;
```
Comment (shaunmaharaj) [inline src/components/appheader.main.less]: Hmm I noticed this now will overlap with the logo on a search. 
image

I proposed something to the designer based on this and waiting on feedback from them. I'm proposing the `search` button click triggering the dropdown to expand, to reveal the search form in the dropdown:
image

Comment (rostyk-kanafotskyy) [reply src/components/appheader.main.less]: Thank Shaun for pointing out, we will fix this issue tomorrow morning.

Comment (shaunmaharaj) [reply src/components/appheader.main.less]: Hey @rostyk-kanafotskyy! I got a reply back from the designer and the alternative is approved for implementation. The search button click can trigger the dropdown to expand, then put focus on the form field in the dropdown. Thank you!

**Pull request merged**